src/test/java/iudx/resource/server/database/archives/DatabaseServiceTest.java

```diff
@@ -2101,7 +2101,9 @@ void searchAttribute3(VertxTestContext testContext) {
         .search(request)
         .onFailure(
             handler -> {
-              assertEquals(handler.getMessage(),"Result Limit exceeds");
+              assertEquals(
+                  handler.getMessage(),
+                  "{\"type\":413,\"title\":\"urn:dx:rs:payloadTooLarge\",\"details\":\"Response size exceeds limit\"}");
```

Comment:
Create a JsonObject instead of a String.

Reply:
Solved